libafl/src/generators/mod.rs

```diff
@@ -74,64 +73,45 @@ where
 
 #[derive(Clone, Debug)]
 /// Generates random bytes
-pub struct RandBytesGenerator<S> {
+pub struct RandBytesGenerator {
     max_size: NonZeroUsize,
-    phantom: PhantomData<S>,
 }
 
-impl<S> Generator<BytesInput, S> for RandBytesGenerator<S>
+impl<S> Generator<BytesInput, S> for RandBytesGenerator
 where
     S: HasRand,
 {
     fn generate(&mut self, state: &mut S) -> Result<BytesInput, Error> {
         let mut size = state.rand_mut().below(self.max_size);
-        if size == 0 {
-            size = 1;
-        }
+        size = min(size, 1);
```

Comment:
Hey @tokatoka, not sure I'm understanding this change. Now the function will generate only ByteInput of 1 byte?

Reply:
yeah sorry i am dumb this should be max